shared/services/config/stader-config.go

```diff
@@ -88,7 +90,7 @@ type StaderConfig struct {
 
 	// Metrics settings
 	EnableMetrics           config.Parameter `yaml:"enableMetrics,omitempty"`
-	ExposeGuardianPort      config.Parameter `yaml:"enableMetrics,omitempty"`
+	ExposeGuardianPort      config.Parameter `yaml:"exposeGuardianPort,omitempty"`
```

Comment:
Why you change this, this's not backward compatible change

Reply:
yea,  you mean for users who have not set exposeGuardianPort config. I see service still start without this config set.

this is something we should fix
the `stader-cli service get-config-yaml` breaks btw

Reply:
if exiting node had set `ExposeGuardianPort`  to `true`, which's the value you load in new code?